docs/develop/python/temporal-client.mdx

```diff
@@ -0,0 +1,300 @@
+keywords:
+  - temporal python client
+  - connect python client to temporal service
+  - initialize temporal client
+  - temporal SDK python guide
+  - start workflow execution python
+  - temporal cloud connection
+  - python client for temporal cli
+  - custom namespace configuration
+  - temporal workflow management
+  - temporal client setup
+  - python workflow execution
+  - temporal cloud integration
+  - temporal client options
+  - managing temporal namespaces
+tags:
+  - python
+  - python-sdk
+  - temporal-client
+---
+
+This guide introduces Temporal Clients.
+It explains the role and use of Clients and shows you how to configure your Python Client code to connect to the Temporal Service.
+
+The pages shows how to do the following:
+
+- [Connect to a local development Temporal Service](#connect-to-development-service)
+- [Connect to Temporal Cloud](#connect-to-temporal-cloud)
+- [Start a Workflow Execution](#start-workflow-execution)
+
+## Connect to development Temporal Service {#connect-to-development-service}
+
+**How to connect to the local Temporal CLI development Temporal Service using the Python SDK**
+
+A [Temporal Client](/encyclopedia/temporal-sdks#temporal-client) enables you to communicate with the [Temporal Cluster](/clusters).
+Communication with a Temporal Cluster includes, but isn't limited to, the following:
+
+- Starting Workflow Executions.
+- Sending Signals to Workflow Executions.
+- Sending Queries to Workflow Executions.
+- Getting the results of a Workflow Execution.
+- Providing an Activity Task Token.
+
+:::caution
+
+A Temporal Client cannot be initialized and used inside a Workflow.
+However, it is acceptable and common to use a Temporal Client inside an Activity to communicate with a Temporal Cluster.
+
+:::
+
+When you are running a Cluster locally (such as the [Temporal CLI](https://docs.temporal.io/cli/server#start-dev)), the number of connection options you must provide is minimal.
+Many SDKs default to the local host or IP address and port that Temporalite and [Docker Compose](https://github.com/temporalio/docker-compose) serve (`127.0.0.1:7233`).
+
+Use the `connect()` method on the Client class to create and connect to a Temporal Client to the Temporal Cluster.
+
+<div class="copycode-notice-container">
+  <a href="https://github.com/temporalio/documentation/blob/main/sample-apps/python/your_app/run_workflow_dacx.py">
+    View the source code
+  </a>{" "}
+  in the context of the rest of the application code.
+</div>
+
+```python
+
+# ...
+async def main():
+    client = await Client.connect("localhost:7233")
+
+    result = await client.execute_workflow(
+        YourWorkflow.run,
+        "your name",
+        id="your-workflow-id",
+        task_queue="your-task-queue",
+    )
+
+    print(f"Result: {result}")
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+## How to connect to Temporal Cloud {#connect-to-temporal-cloud}
```

Comment:
We want to nix the "How to" from the headers so the headers are shorter and rely on the bold first sentence to set the scope i.e.

> **How to connect to Temporal Cloud using the Python SDK**

We will want to update all header